supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesAsyncBackendJobExecutionActor.scala

```diff
@@ -192,16 +193,9 @@ class JesAsyncBackendJobExecutionActor(override val jobDescriptor: BackendJobDes
     new JesExpressionFunctions(List(gcsFileSystem), callContext)
   }
 
-  private lazy val knownInputs = {
-    val unqualifiedWorkflowInputs = workflowDescriptor.inputs map {
-      case (fqn, v) => splitFqn(fqn)._2 -> v
-    }
-    unqualifiedWorkflowInputs ++ jobDescriptor.inputs
-  }
-
   private val lookup: ScopedLookupFunction = {
     val declarations = workflowDescriptor.workflowNamespace.workflow.declarations ++ call.task.declarations
-    WdlExpression.standardLookupFunction(knownInputs, declarations, callEngineFunctions)
+    WdlExpression.standardLookupFunction(jobDescriptor.inputs, declarations, callEngineFunctions)
```

Comment:
This was a deliberate choice. Previously I was getting odd results such as workflow declarations showing up in calls' metadata for inputs despite not being available to the command code.